utils.js

```diff
@@ -0,0 +1,25 @@
+const isDef = v => v !== undefined && v !== null;
+
+function normaliseProjectName(name) {
```

Comment:
This function and file is no longer being used. Please delete

Reply:
done